test/conformance/exp_command_buffer/fixtures.h

```diff
@@ -304,6 +301,128 @@ struct urCommandBufferMultipleKernelUpdateTest
     std::vector<std::shared_ptr<TestKernel>> TestKernels{};
 };
 
+struct urCommandEventSyncTest : urCommandBufferExpTest {
+    void SetUp() override {
+        UUR_RETURN_ON_FATAL_FAILURE(urCommandBufferExpTest::SetUp());
+
+        ur_bool_t event_support = false;
+        ASSERT_SUCCESS(urDeviceGetInfo(
+            device, UR_DEVICE_INFO_COMMAND_BUFFER_EVENT_SUPPORT_EXP,
+            sizeof(ur_bool_t), &event_support, nullptr));
+        if (!event_support) {
+            GTEST_SKIP() << "External event sync is not supported by device.";
+        }
+
+        ur_queue_flags_t flags = UR_QUEUE_FLAG_SUBMISSION_BATCHED;
+        ur_queue_properties_t props = {
+            /*.stype =*/UR_STRUCTURE_TYPE_QUEUE_PROPERTIES,
+            /*.pNext =*/nullptr,
+            /*.flags =*/flags,
+        };
+        ASSERT_SUCCESS(urQueueCreate(context, device, &props, &queue));
+        ASSERT_NE(queue, nullptr);
+
+        for (auto &device_ptr : device_ptrs) {
+            ASSERT_SUCCESS(urUSMDeviceAlloc(context, device, nullptr, nullptr,
+                                            allocation_size, &device_ptr));
+            ASSERT_NE(device_ptr, nullptr);
+        }
+
+        for (auto &buffer : buffers) {
+            ASSERT_SUCCESS(urMemBufferCreate(context, UR_MEM_FLAG_READ_WRITE,
+                                             allocation_size, nullptr,
+                                             &buffer));
+            ASSERT_NE(buffer, nullptr);
+        }
+
+        // Create a command-buffer with update enabled.
+        ur_exp_command_buffer_desc_t desc{
+            UR_STRUCTURE_TYPE_EXP_COMMAND_BUFFER_DESC, nullptr, true};
```

Comment:
It is a bit confusing that this fixture is does not mention updates and creates an updatable command buffe. Does it have to be marked as updatable?

Reply:
Good catch! This was just an oversight. Updated to fully initalize the struct rather than only some of the members, and make updatable false, i.e.
```cpp
         ur_exp_command_buffer_desc_t desc{
            /*.stype=*/UR_STRUCTURE_TYPE_EXP_COMMAND_BUFFER_DESC,
            /*.pNext =*/nullptr,
            /*.isUpdatable =*/false,
            /*.isInOrder =*/false,
            /*.enableProfiling =*/false,
        };
```